devops/docker/smqtk_services/Dockerfile

```diff
@@ -54,23 +54,25 @@ RUN cd /caffe/build \
     -DUSE_CUDNN:BOOL=OFF \
     -DCMAKE_INSTALL_PREFIX:PATH=/caffe/install \
     /caffe/source \
- && make install -j12 \
+ && make install -j2 \
```

Comment:
for whatever reason, the build kept hanging on my Mac 
```
MacBook Pro (13-inch, 2017, Four Thunderbolt 3 Ports)
Processor: 3.5 GHz Intel Core i7
Mac OS X High Sierra 10.13.4
Docker version 18.03.0-ce, build 0520e24
```
without changing this to something smaller.

Reply:
This is probably pinned to some old version of caffe that was known to work at the time (their versioning was pretty bad).  Depending on whether this is a hash before or after rc3, the most recent version tag (1.0) will probably work and be more stable than this version (there was an API breaking change to Net class constructor around rc3, or 4, I keep forgetting).

Reply:
Should we just replace all these `make -j` portions with `make -j$(nproc)`?

Reply:
+1 from me.

Reply:
(for using nproc)

Reply:
+1 from me too @danlamanna and @Purg to replacing with make -j$(nproc)